ambry-router/src/test/java/com/github/ambry/router/TtlUpdateManagerTest.java

```diff
@@ -390,6 +402,7 @@ private void sendRequestsGetResponses(FutureResult<Void> futureResult, TtlUpdate
         switch (type) {
           case TtlUpdateRequest:
             manager.handleResponse(responseInfo);
+            responseInfo.release();
```

Comment:
Thanks for fixing this, this is all good, But can you move the release to the outside of the for loop. Something like
```
for (ResponseInfo repsonseInfo: responseInfoList) {
}
responseInfoList.forEach(ResponseInfo::release);
```

Reply:
Good suggestion. Changed.